src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,67 @@
+package duke.command;
+
+import duke.Deadline;
+import duke.DukeException;
+import duke.Event;
+import duke.Task;
+import duke.TaskList;
+import duke.TaskType;
+import duke.ToDo;
+import duke.Ui;
+
+public class AddCommand implements Command {
+    private String[] params;
+
+    public AddCommand(String[] params) {
+        this.params = params;
+    }
+
+    public TaskType getTaskType(String tasktypeInput) {
+        return tasktypeInput.equals("event") ? TaskType.Event
+                : tasktypeInput.equals("deadline") ? TaskType.Deadline
+                : tasktypeInput.equals("todo") ? TaskType.ToDo
+                : TaskType.Invalid;
```

Comment:
Would an `if else` statement help with clarity instead?

Reply:
Fixed by removing the function! 🙂